src/schemaGenerator.ts

```diff
@@ -162,7 +171,7 @@ function forEachField(schema, fn) {
 // the connectors to the context by wrapping each query or mutation resolve
 // function with a function that attaches connectors if they don't exist.
 // attaches connectors only once to make sure they are singletons
-function attachConnectorsToContext(schema, connectors) {
+function attachConnectorsToContext(schema: GraphQLSchema, connectors: IConnectors): void {
```

Comment:
Let's put a deprecation warning in this function and tell people not to use it any more. Then we can drop it in the next minor release.